extensions/machine-learning/src/views/widgets/dashboardWidget.ts

```diff
@@ -405,37 +405,22 @@ export class DashboardWidget {
 			width: maxWidth + 10,
 			justifyContent: 'flex-start'
 		}).component();
-		const linkComponent = view.modelBuilder.hyperlink().withProperties({
-			label: linkMetaData.title,
-			url: linkMetaData.link,
+		const linkComponent = view.modelBuilder.hyperlink().withProps({
```

Comment:
why change to withProps? I thought withProps and withProperties were synonyms, are they not?

Reply:
when you use withProperties without provide <T>, you can pass in any value, but withProps is strong typed.